src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java

```diff
@@ -0,0 +1,34 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.testng.annotations.Test;
+
+public class TestHorseSlots extends NewTestTemplate {
+
+  @Test(groups = {"TestHorseSlots_FloorAdhesion", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_FloorAdhesion() {
+    driver.manage().window().maximize();
+    driver.get("http://aga.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/FloorAdhesion");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("google_ads_iframe_/5441/wka.gaming/_aga//article/gpt/INVISIBLE_HIGH_IMPACT_2_0__container__");
+    try {
+      Thread.sleep(10000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)